src/main/java/com/denizenscript/denizencore/objects/core/ListTag.java

```diff
@@ -485,39 +483,37 @@ else if (brackets == 0 && chr == '|') {
     }
 
     public ListTag(ListTag input) {
-        objectForms = new ArrayList<>(input.objectForms);
+        this(input.objectForms);
     }
 
     public ListTag(List<String> items, boolean isPlainText) {
-        objectForms = new ArrayList<>(items.size());
-        for (String str : items) {
-            objectForms.add(new ElementTag(str, isPlainText));
-        }
+        this(items, str -> new ElementTag(str, isPlainText));

```

Comment:
You replaced good clean code with a more complicated and lower performance method? 0.o